packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -24,7 +24,7 @@ export function buildPlugin({
   onEnd = () => {},
 }: Options) {
   return build({
-    entryPoints: [resolve(__dirname, "../pages/functions/template-plugin.ts")],
+    entryPoints: [resolve(__dirname, "../templates/template-plugin.ts")],
```

Comment:
Be careful you don't fall foul of issues related to https://github.com/cloudflare/wrangler2/pull/860 here.